server/src/main/java/org/opensearch/cluster/routing/allocation/AllocationService.java

```diff
@@ -183,9 +200,9 @@ private ClusterState buildResult(ClusterState oldState, RoutingAllocation alloca
         if (restoreInProgress != null) {
             RestoreInProgress updatedRestoreInProgress = allocation.updateRestoreInfoWithRoutingChanges(restoreInProgress);
             if (updatedRestoreInProgress != restoreInProgress) {
-                ImmutableOpenMap.Builder<String, ClusterState.Custom> customsBuilder = ImmutableOpenMap.builder(allocation.getCustoms());
+                final Map<String, ClusterState.Custom> customsBuilder = new HashMap<>(allocation.getCustoms());
                 customsBuilder.put(RestoreInProgress.TYPE, updatedRestoreInProgress);
-                newStateBuilder.customs(customsBuilder.build());
+                newStateBuilder.customs(customsBuilder);
```

Comment:
Not sure how this diff got its way, Will remove.

Reply:
@Gaurav614  - is it resolved ?